math/vecops/inc/ROOT/RVec.hxx

```diff
@@ -1880,6 +1880,11 @@ T Sum(const RVec<T> &v, const T zero = T(0))
    return std::accumulate(v.begin(), v.end(), zero);
 }
 
+inline std::size_t Sum(const RVec<bool> &v, const std::size_t zero = 0ul)
```

Comment:
What is the reason to add the `const` qualifier to the second argument?

Reply:
it was just a copy-paste from the templated version, I removed the `const`, thanks

Reply:
Maybe even the templated version should have `const T&` as second argument type?